examples/ciao/README.md

```diff
@@ -42,7 +50,7 @@ to install the `qemu` package
 ### ansible-galaxy
 Install the required roles from ansible-galaxy
 
-    ansible-galaxy install -r requirements.yml
+    ansible-galaxy install --ignore-certs -r requirements.yml
```

Comment:
Why was the --ignore-certs needed?  Was this because I was running ansible behind a proxy?  Is there any way to avoid this?  In general we probably shouldn't be advising people to ignore certs.  So my request is:

1. Figure out why we need the --ignore-certs.
2. Check to see if there is an ansible bug open on this.
3. Assuming it is only needed if we are behind a proxy mention this.  We don't want people to use this flag if it's not needed.